sources/components/custom-styles/template-cs-style-rules.scss

```diff
@@ -984,3 +989,107 @@ body {
     text-transform: var(--content-area-product-title__text-transform);
   }
 }
+
+.dark-background {
+  .edy-product-widget-grid .edy-product-widget-item {
+    .edy-product-widget-item-details-wrap {
+      .edy-product-widget-item-title,
+      .edy-product-widget-item-link {
+        color: var(--third-color);
+      }
+
+      .edy-product-widget-item-price,
+      .edy-product-widget-item-description {
+        color: var(--third-color);
+        opacity: .7;
+      }
+    }
+
+    .edy-product-widget-item-top-outer .edy-product-widget-item-top.edy-product-widget-item-without-image {
+      border: 1px solid var(--third-color);
+      opacity: .5;
+    }
+
+    .edy-product-widget-item-details-flex-wrap {
+      .edy-product-widget-item-out-of-stock,
+      .edy-product-widget-item-btn {
+        color: var(--third-color);
+      }
+    }
+    
+    &:hover {
+      .edy-product-widget-item-price {
+        opacity: 0;
+      }
+    }
+  }
+}
+
+.edy-product-widget-list .edy-product-widget-item-wrap .edy-product-widget-item .edy-product-widget-item-details-wrap {
+  .dark-background & {
+    .edy-product-widget-item-header {
+      .edy-product-widget-item-title {
+        .edy-product-widget-item-link {
+          color: var(--third-color);
+        }
+      }
+
+      .edy-product-widget-item-price {
+        color: var(--third-color);
+        opacity: .7;
+      }
+
+      .edy-product-widget-item-out-of-stock {
+        color: var(--third-color);
+      }
+    }
+    
+    .edy-product-widget-item-description {
+      color: var(--third-color);
+    }
+
+    .edy-product-widget-item-without-image {
+      border: 1px solid var(--third-color);
+      opacity: .5;
+    }
+  }
+
+  .edy-product-widget-item-btn-wrap .edy-product-widget-item-btn {
+    .light-background &,
+    .dark-background & {
+      padding: calc(var(--button-padding) - 18px) var(--button-padding) calc(var(--button-padding) - 17px);
```

Comment:
The odd (17px) )value seems a little dubious.

Reply:
Regular button uses the same value, for consistency perhaps let's keep it the same
https://github.com/Voog/design-anchorage/blob/6a8fe7eed53647a8518e7c6893dad3a7de4eaa58/sources/components/custom-styles/template-cs-style-rules.scss#L818

Reply:
Ok.